src/wp-includes/default-filters.php

```diff
@@ -748,4 +748,9 @@
 // Font management.
 add_action( 'wp_head', 'wp_print_font_faces', 50 );
 
+// Modules
+add_action( 'wp_head', array( 'WP_Modules', 'print_import_map' ) );
+add_action( 'wp_head', array( 'WP_Modules', 'print_enqueued_modules' ) );
+add_action( 'wp_head', array( 'WP_Modules', 'print_module_preloads' ) );
```

Comment:
It would be great if you could introduce wrapper functions for these 3 as well (i.e. `wp_print_import_map()` etc.). This way you can reference the functions here, which avoids loading/instantiating the class early in the WordPress bootstrap process where it may not be needed.

Not a big deal, but an easy enough optimization IMO worth making.

Reply:
Sure. Done in https://github.com/WordPress/wordpress-develop/pull/5818/commits/eaaa563b214fd1bcd2b16642f6e628554711748b 🙂